public/manifest.json

```diff
@@ -1,8 +1,8 @@
 {
-  "manifest_version": 3,
+  "manifest_version": 4,
```

Comment:
The manifest version shouldn't change: https://developer.chrome.com/docs/extensions/reference/manifest/manifest-version